katalog/etcd-backup-pvc/README.md

```diff
@@ -0,0 +1,129 @@
+# ETCD Backup PVC
+
+This package provides an automated solution for backing up ETCD data to a Persistent Volume Claim present in the cluster.
+
+## Overview
+
+This package creates a CronJob that:
+
+1. Takes a snapshot of the ETCD database
+2. Copies the snapshot to a defined PersistentVolumeName
+3. Cleans up old backups based on a configurable retention policy
+
+The job is scheduled to run on one of the available control plane nodes and uses `rclone` to manage the actual backup and its lifecycle.
+
+The job is fault-tolerant, as it tries to backup the available etcd nodes:
+even if one of the etcd nodes (if you're in a HA-setup) is failing,
+`etcd-backup-pvc` asks the cluster which nodes are healthy and spashots one of
+them.
+
+> [!NOTE]
+> The job is configured to use the Host Network, as a consequence it cannot resolve service names internal to the cluster or connect to them via the internal Kubernetes network.
+
+## Requirements
+
+- A Kubernetes cluster with ETCD
+- Access to control plane nodes
+- A pre-provisioned PersistentVolumeClaim
+
+> [!NOTE]
+> By default we rely on `ClusterConfiguration` found inside the `kubeadm-config` ConfigMap to
+> extract the ETCD server endpoints. If you want to use a cluster that's not managed by
+> `kubeadm`, please make sure to remove the `kubeadm-config` volume, and pass the CronJob the
+> `ETCDCTL_ENDPOINTS` environment variable set accordingly.
+
+## Configuration
+
+### Default Configuration
+
+The package includes the following default configuration:
+
+- The backup runs on specified schedule (configurable)
+- Snapshots are stored in the format `fury-etcd-snapshot-YYYYMMDDHHMM.etcdb`
```

Comment:
Maybe we should not add `fury` to the name, being that we are about to start a renaming.

could be a good idea to put the cluster name in the filename instead of `fury` or maybe inside a folder with the cluster name?. Could it be that someone wants to use the same PVC / Bucket for more than 1 cluster backup? Just a thought, we can leave that for a v2 eventuallly.

Reply:
Yes, that could be a perfectly fine idea. In fact the name should be somehow configurable.

Reply:
I now made the name configurable, but this requires https://github.com/sighupio/fury-distribution-container-image-sync/pull/317 this PR to be merged before it's available.